<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2458</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="table">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="CopyButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Duplicates a row in the table</string>
         </property>
         <property name="text">
          <string>Duplicate Row</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="DeleteButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Deletes selected row</string>
         </property>
         <property name="text">
          <string>Delete Row</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="DeleteAllButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Deletes all rows in the table</string>
         </property>
         <property name="text">
          <string>Delete all Rows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QPushButton" name="MarkCurrentETLParametersButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies current ETL values into the selected row.</string>
         </property>
         <property name="text">
          <string>Mark ETL values</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="PreviewSelectionButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Preview acquisition: The microscope will move the to the starting location and set up everything (filters, lasers, ...).</string>
         </property>
         <property name="text">
          <string>Preview Selected Row</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="ImageProcessingWizardButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Generates Image Processing Options for the entire table.</string>
         </property>
         <property name="text">
          <string>Image Processing Wiz</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="SaveButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Saves the table</string>
         </property>
         <property name="text">
          <string>Save Table</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QPushButton" name="MarkCurrentRotationButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies current sample rotation angle into the selected row.</string>
         </property>
         <property name="text">
          <string>Mark Rotation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="MoveUpButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Moves selected row up</string>
         </property>
         <property name="text">
          <string>Move Row Up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="AddButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Adds another row to the table</string>
         </property>
         <property name="text">
          <string>Add Row</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QPushButton" name="MarkCurrentFocusButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies the current focus into the selected row (Sets both start &amp; end focus).</string>
         </property>
         <property name="text">
          <string>Mark Focus</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="MarkCurrentXYButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies the current X &amp; Y position into the selected row.</string>
         </property>
         <property name="text">
          <string>Mark Current XY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="FocusTrackingWizardButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Generates start/end focus positions for the table.</string>
         </property>
         <property name="text">
          <string>Focus Tracking Wiz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QPushButton" name="MarkCurrentStateButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies current laser, intensity, zoom &amp; filter settings into the selected row.</string>
         </property>
         <property name="text">
          <string>Mark State</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="MoveDownButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Moves selected row down</string>
         </property>
         <property name="text">
          <string>Move Row Down</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="LoadButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Loads a saved table.</string>
         </property>
         <property name="text">
          <string>Load Table</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="FilenameWizardButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Generates filenames for the entire table.</string>
         </property>
         <property name="text">
          <string>Filename Wiz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="PreviewZCheckBox">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>If enabled, the sample will be moved to the z-starting location if &quot;Preview Selected Row&quot; is clicked.</string>
         </property>
         <property name="text">
          <string>Preview: Move to Z start</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="AcquisitionTimeLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Predicted acquisition time in hh:mm:ss.</string>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="TilingWizardButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Generates a tiling acquisition through a series of steps.</string>
         </property>
         <property name="text">
          <string>Tiling Wizard</string>
         </property>
        </widget>
       </item>
       <item row="2" column="9">
        <widget class="QPushButton" name="MarkAllButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Copies all current settings into the selected row.</string>
         </property>
         <property name="text">
          <string>Mark ALL</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Predicted acquisition time in hh:mm:ss, based framerate in the config file.</string>
         </property>
         <property name="text">
          <string>Estimated time &amp; size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="PredictedSizeLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Predicted data size</string>
         </property>
         <property name="text">
          <string>0 G</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="SetFoldersButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Sets all the save folders for the entire table.</string>
         </property>
         <property name="text">
          <string>Set Folders</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QPushButton" name="AutoIlliminationButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Set the illumination direction (Left/Right) of edge tiles based on their position (minimize light-sheet travel and scattering).</string>
         </property>
         <property name="text">
          <string>Auto L/R illumination</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
